src/v2/view-builder/components/IdentifierFooter.js

```diff
@@ -6,6 +6,9 @@ import Link from './Link';
 import hbs from 'handlebars-inline-precompile';
 
 export default BaseFooter.extend({
+
+  shouldShowForgotPasswordLink: () => !this.options.appState.isIdentifierOnlyView(),
```

Comment:
just `showForgotPassworkLink` or `hasForgotPassworkLink can work.. we can remove `should`

Reply:
done